test/Microsoft.AspNetCore.Server.Kestrel.Core.Tests/FrameResponseHeadersTests.cs

```diff
@@ -59,14 +64,14 @@ public void InitialDictionaryIsEmpty()
         [InlineData("Ser\u0080ver", "Data")]
         [InlineData("Server", "Da\u0080ta")]
         [InlineData("Unknown\u0080-Header", "Data")]
-        [InlineData("Ser�ver", "Data")]
-        [InlineData("Server", "Da�ta")]
-        [InlineData("Unknown�-Header", "Data")]
-        [InlineData("Ser�ver", "Data")]
-        [InlineData("�erver", "Data")]
-        [InlineData("Server", "Da�ta")]
-        [InlineData("Unknown�-Header", "Data")]
-        [InlineData("Ser�ver", "Data")]
+        [InlineData("Ser™ver", "Data")]
+        [InlineData("Server", "Da™ta")]
+        [InlineData("Unknown™-Header", "Data")]
+        [InlineData("Ser™ver", "Data")]
+        [InlineData("šerver", "Data")]
+        [InlineData("Server", "Dašta")]
+        [InlineData("Unknownš-Header", "Data")]
+        [InlineData("Seršver", "Data")]
```

Comment:
File -> Save As... > Save 🔽  > Save with Encoding...

Actually, I suspect the new encoding is right, but good to double check that its utf8.

![image](https://cloud.githubusercontent.com/assets/2696087/25461924/967f0da6-2aa0-11e7-960c-5e67d94d8aa0.png)